en/components/sprite.md

```diff
@@ -23,7 +23,7 @@ Please refer to [Sprite API](../../../api/en/classes/Sprite.html) for the script
 | Src Blend Factor | Current Image Blending Mode
 | Dst Blend Factor | Background image blend mode, combined with the above properties, can blend foreground and background Sprite in different ways. Preview of effect can refer to [glBlendFunc Tool](http://www.andersriggelsen.dk/glblendfunc.php)
 
-After adding the **Sprite** component, you can display the resource image through the Sprite component by dragging a resource of type **Texture** or **SpriteFrame** from the **Explorer** into the **Sprite Frame** property reference.
+After adding the **Sprite** component, you can display the resource image through the Sprite component by dragging a resource of type **Texture** or **SpriteFrame** from the **Assets Panel** into the **Sprite Frame** property reference.
```

Comment:
I think it should be: 

![image](https://user-images.githubusercontent.com/19702899/49193165-187acb00-f3b9-11e8-903a-b1e7a6068ebf.png)